package.json

```diff
@@ -81,7 +82,8 @@
     "web_modules/"
   ],
   "scripts": {
-    "pretest": "npm run lint && npm run beautify-lint",
+    "lint-file-standard": "eclint check lib/**/*.js hot/*.js bin/*.js benchmark/*.js test/*.js",
+    "pretest": "npm run lint && npm run beautify-lint && npm run lint-file-standard",
```

Comment:
Sure. Good catch! I added the related changes in [this commit](https://github.com/webpack/webpack/pull/2849/commits/3f3b7800fda9334eaedab69754f1553a4c6f7567)